src/dmd/astbase.d

```diff
@@ -3633,15 +3636,35 @@ struct ASTBase
         }
     }
 
-    extern (C++) final class TypeClass : Type
+    extern (C++) abstract class TypeAggregate : Type
     {
-        ClassDeclaration sym;
+        AggregateDeclaration asym;
+        alias sym = asym;
```

Comment:
Why is this alias necessary? Can't you simply do:

```d
AggregateDeclaration sym;
```

?

Reply:
It conflicts with the sym () property on child nodes. But I can't think of a better alternative